### ISSUE TITLE
Remove "sticky-post" from tag list

### ISSUE THREAD

**Issue by nielslange** [WordPress/twentytwenty#739]
See the discussion from https://github.com/WordPress/twentytwenty/issues/373#issuecomment-537969092 on.

**Comment by ianbelanger79** [comment]
Thanks for submitting this @nielslange, but the team feels that the theme clearly does support `sticky-posts` therefor, the tag will stay.

**Comment by nielslange** [comment]
Sounds good to me, @ianbelanger79. However, we should then adjust the visual presentations of sticky posts, as mentioned by @joyously in https://github.com/WordPress/twentytwenty/issues/373#issuecomment-537982296. 

@andersnoren Can you create another issue to improve the visual presentations of sticky posts or do you have some other CSS improvements in which you can adjust the sticky posts styles in parallel?

**Comment by joyously** [comment]
> but the team feels that the theme clearly does support sticky-posts therefor, the tag will stay.

Can you link to where this was decided? It clearly does not match the Theme Requirements for that Tag.
